### DISCUSSION TITLE
import data values from execl file

### DISCUSSION THREAD

**Discussion by abdelaziz009** [cotes2020/jekyll-theme-chirpy#2098] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### Is your feature request related to a problem? Please describe

i want to create a post containg table with serach feature and the values are from excel sheet with copied under assets/data and i added a line to to the dockr file to copy this file to the container while build but unfortunatlly only the table header is showing the values not showing, i tried with .cvs & .xlsx also but didnt succeed 

### Describe the solution you'd like

noting that i surved the site locally outside "a container" and the table values was showing normally i dont know what is the issue

### Describe alternatives you've considered

simple database

### Additional context

-

**Comment by kungfux** [comment]
This does not look like a bug report, but a community question. Let me move it to the discussion for you.

**Comment by abdelaziz009** [comment]
thanks you @kungfux , seeking the community support
